resources/docs/livewire/editing-chirps.md

```diff
@@ -0,0 +1,517 @@
+[TOC]
+
+# <b>05.</b> Editing Chirps
+
+Let's add a feature that's missing from other popular bird-themed microblogging platforms &mdash; the ability to edit Chirps!
```

Comment:
Unrelated to this PR, but I wonder whether this should be updated now that you can edit on the platform we're referencing (with a subscription). We could still be playful - perhaps mentioning that we'll let our users do it for free, but then maybe mentioning Spark as a way to create subscriptions that unlock features.

I also wonder if we should remove the "bird-themed" reference from here, as it's not as relevant any more. We could instead update the introduction page to reference the rename and keep the "Chirper" name relevant by saying something like "We miss having a bird-themed micro-blogging platform in our lives, so let's create a new one called Chirper".

Reply:
Lets address this issue on a different PR, as it touches all stacks and the introduction of this bootcamp.